PINCache/PINOperationQueue.m

```diff
@@ -128,18 +128,33 @@ - (void)dealloc
   return reference;
 }
 
+- (void)cancelAllOperations
+{
+  [self lock];
+  for (PINOperation *operation in [_referenceToOperations objectEnumerator]) {
```

Comment:
Another advantage of the _referenceToOperations weakTable :) We wouldn't have been able to enumerate over _queuedOperations without making a copy. Do you know if we need a copy here? Is it safe to enumerate over it even as objects are potentially being nil'd out by locked_cancelOperation?